integration-tests/src/tests/client/resharding.rs

```diff
@@ -605,6 +606,28 @@ impl TestReshardingEnv {
             }
         }
     }
+
+    fn check_snapshot(&mut self, state_snapshot_enabled: bool) {
```

Comment:
In the previous PR https://github.com/near/nearcore/pull/9681, we added a test to check if resharding happens when state snapshot config is enabled or disabled. In this PR we extend the tests to check whether the snapshot is deleted or not.

Reply:
what does 'check_snapshot' mean? (check 'What' of snapshot?) function name doesn't sound clear enough.

Reply:
I'm fine with the function name as it's similar to others but please add a comment :)